xline/src/server/lease_server.rs

```diff
@@ -363,11 +381,9 @@ where
                 )
             });
             if !self.lease_storage.is_primary() {
-                let mut lease_client = LeaseClient::connect(format!("http://{leader_addr}"))
-                    .await
-                    .map_err(|e| {
-                        tonic::Status::internal(format!("Connect to leader error: {e}"))
-                    })?;
+                let endpoints = build_endpoints(leader_addr)?;
```

Comment:
Please use plural `leader_addrs`. And update method naming in `ClusterInfo`.

Reply:
done